MapboxNavigation/JunctionView.swift

```diff
@@ -7,6 +7,7 @@ import MapboxDirections
 public class JunctionView: UIImageView {
     var isCurrentlyVisible: Bool = false
     var imageRepository: ImageRepository = .shared
+    var distanceAlongStep: CLLocationDistance = -1
```

Comment:
Make this property optional to avoid magic numbers like −1.

Reply:
👍